deploy/backplane/lpsre/acm/02-acm-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -0,0 +1,959 @@
+  - hiveconfigs
+  - ingresses
+  - managedclusters
+  - managedclustersets
+  - multiclusterhubs
+  - namespaces
+  - pods
+  - policyreports
+  - rolebindings
+  - secrets
+  - services
+  - subjectaccessreviews
+  - validatingwebhookconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ''
+  - admissionregistration.k8s.io
+  - apps
+  - apps.open-cluster-management.io
+  - mcm.ibm.com
+  - monitoring.coreos.com
+  - operator.open-cluster-management.io
+  resources:
+  - deployments
+  - deployments/finalizers
+  - helmreleases
+  - multiclusterhubs
+  - multiclusterhubs/finalizers
+  - multiclusterhubs/status
+  - servicemonitors
+  - servicemonitors/finalizers
+  - services
+  - services/finalizers
+  - validatingwebhookconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ''
+  - apiextensions.k8s.io
+  - apiregistration.k8s.io
+  - apps
+  - apps.open-cluster-management.io
+  resources:
+  - apiservices
+  - channels
+  - customresourcedefinitions
+  - deployments
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ''
+  - apps
+  resources:
+  - deployments
+  - serviceaccounts
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - admissionregistration.k8s.io
+  - apiextensions.k8s.io
+  - apiregistration.k8s.io
+  - hive.openshift.io
+  - mcm.ibm.com
+  - rbac.authorization.k8s.io
+  resources:
+  - apiservices
+  - clusterrolebindings
+  - clusterroles
+  - customresourcedefinitions
+  - hiveconfigs
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - agent-install.openshift.io
+  resources:
+  - agentserviceconfigs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - console.openshift.io
+  - search.open-cluster-management.io
+  resources:
+  - consolelinks
+  - consoleplugins
+  - searches
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - prometheusrules
+  - servicemonitors
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - multicluster.openshift.io
+  resources:
+  - multiclusterengines
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - operator.openshift.io
+  resources:
+  - consoles
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - clusterserviceversions
+  - operatorgroups
+  - subscriptions
+  verbs:
+  - delete
+  - get
+  - list
+  - watch
+- apiGroups:
+  - packages.operators.coreos.com
+  resources:
+  - packagemanifests
+  verbs:
+  - get
+  - list
+  - watch
+  serviceAccountName: multiclusterhub-operator
+- rules:
+- apiGroups:
+  - ''
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  - serviceaccounts
+  - namespaces
+  - nodes
```

Comment:
This is bound to the `redhat-open-cluster-management` namespace so these resources do not make sense to include.